src/test/java/ua/antibyte/analyzer/service/impl/CsvReadingServiceImplTest.java

```diff
@@ -0,0 +1,47 @@
+package ua.antibyte.analyzer.service.impl;
+
+import com.opencsv.CSVReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import ua.antibyte.analyzer.service.CsvReadingService;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
avoid star imports and place static imports before non-static (the groups should be separated with a single empty line)